src/app/pages/FastBtcPage/components/Withdraw/AddressForm.tsx

```diff
@@ -46,22 +48,37 @@ export const AddressForm: React.FC = () => {
     [set, value],
   );
 
-  const validate = useCallback(async (adr: string) => {
+  const validateAddr = useCallback(async (adr: string) => {
```

Comment:
```suggestion
  const validateAddress = useCallback(async (address: string) => {
```

Reply:
Please, don't use abbreviations when not necessary (we have it in our [Code guidelines](https://sovryn.monday.com/docs/2126716352)).